docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -435,6 +435,49 @@ incorporate the processes associated with crime reporting, the offense-trial
 pipeline, and an awareness of how the relationship between various social actors and
 the algorithm may impact the intended outcomes of a given model.
 
+
+The 4/5ths Rule: A Common Trap
+------------------------------
+
+In the literature around fairness in machine learning, one will often find
+the so-called "4/5ths rule" or "80% rule" used to assess whether a model
+(or mitigation technique) has produced a 'fair' result.
+Typically, the rule is implemented by using the demographic parity ratio introduced
+in the :ref:`disparity_metrics` section, with a result considered 'fair' if the ratio
+exceeds 80% for all identified subgroups.
+*Application of this threshold is wrong in many scenarios.*
+
+**What exactly does the rule mean?**
+Legally, the 80% threshold is a limit for
+`prima facie evidence <https://en.wikipedia.org/wiki/Prima_facie>`_
+that illegal discrimination has occurred relative to a chosen,
+relevant control population.
+A causal link between the statistic and the alleged discrimination must also
+be demonstrated.
+However, the 'Four-Fifths Rule' does not prove or disprove any case.
+It is possible to argue successfully that a demographic parity
+ratio of 50% is still fair.
+It is equally possible to argue that even at a 95% ratio, illegal
+discrimination has occurred.
+Conflating the *disparate impact metric* with the *legal finding of disparate impact*
+is a form of epistemic trespassing.
+
+**Where is the 4/5ths rule valid?**
+The answer is that it is only valid within a specific area of US
+federal employment law.
+Outside this context, the rule has no particular claim to validity.
+
+Taken together, we see that applying the 4/5ths rule will
+not be appropriate in most cases.
+Even in cases where it is appropriate, it does not automatically
+avoid legal jeopardy, much less ensure that results are fair.
+
```

Comment:
I think we can add to this paragraph that this type of situation is why we try to avoid legal terminology as much as possible in Fairlearn, which explains why we do not call it "disparate impact" but "demographic parity".

Reply:
Done.